### ISSUE TITLE
Dark theme - State of active option in options menu component lacks 3 to 1 contrast ratio

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#575]
![image](https://user-images.githubusercontent.com/3180185/159585937-03afd71b-aaaf-47be-b542-5cd6ec8bccc0.png)

---
<dl><dt>Description</dt><dd> The state of an ACTIVE user interface component does not have at least a 3 to 1 luminosity contrast ratio with either the inner or outer adjacent background.
Dark theme - The focus indicator color of the options present in the "Options Menu" component lacks a 3 to 1 contrast ratio.

Focus Indicator color:#EB94FF
Adjacent color:#FFFFFF
The contrast ratio is: 2.1:1</dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
The visual state of an active user interface component MUST have sufficient contrast of 3 to 1 with the adjacent background. Exceptions exist.

HOW TO FIX: 
Fix this issue by adjusting the visual focus indicator or state indicator (e.g. selected, checked, pressed, etc.) of the user interface component and/or background to increase the contrast with either the inner or outer adjacent background to at least 3 to 1.

REFERENCE: 
WCAG Understanding document: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast

BACKGROUND: 
People who have low vision or are colorblind may have difficulty perceiving that an element is interactive or what its state is - e.g. whether it has keyboard focus, is selected/checked/pressed, etc. - if the contrast between the element boundaries and/or state indicators and its background or adjacent colors is insufficient. When an interactive element's visual boundaries and state indicators have adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive which elements on the screen are interactive and what their current state is.</dd><dt>Source code</dt><dd>

```html
<button type="button" aria-expanded="false">Menu</button>
```
</dd><dt>Checkpoint</dt><dd>Non-Text Contrast - States of User Interface Components (1.4.11.b)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/OptionsMenu</dd></dl>

**Comment by schne324** [comment]
@bobbyomari we'll need new color(s) to use here. Please advise

**Comment by scurker** [comment]
I believe this ticket it out of date, we now have a "dark mode" options menu:

![options menu in dark mode](https://user-images.githubusercontent.com/1062039/219764075-94f3cd1a-9f52-4638-8944-3abd4b6525bb.png)

The two colors (`#283640`/`#283640`) have a contrast ratio of 6.03:1.
